Workflow/Sources/SubtreeManager.swift

```diff
@@ -315,10 +313,7 @@ extension WorkflowNode.SubtreeManager {
                 fatalError("[\(WorkflowType.self)] Sink sent an action inside `render`. Sinks are not valid until `render` has completed.")
 
             case .pending:
-                validationState = .queued(event)
-
-            case .queued:
-                fatalError("[\(WorkflowType.self)] Action sent to pipe while already in the `queueing` state.")
+                fatalError("[\(WorkflowType.self)] Action sent to pipe while in the `pending` state.")
```

Comment:
I thought this PR was to eliminate this error?

Reply:
Yeah, this should not be possible, however the `.pending` state will continue to exist. `fatal` ensures we don't hit this spot.